From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rachapalli Bandi Jagadeesh <jagadeesh.rachapalli.bandi@intel.com>
Date: Thu, 30 Oct 2025 11:56:39 +0530
Subject: [PATCH] disable dependancy on PAGE_SIZE_4KB config

PAGE_SIZE_4KB config dependancy has been added for XE driver to support
only 4KB page size to avoid issues with higher page sizes.
Since X86 bu defualt supports 4KB page size, it's safe to remove this
config check.

Reference:
	022906afdf90 Mark xe driver as BROKEN if kernel page size is not 4kB

Signed-off-by: Rachapalli Bandi Jagadeesh <jagadeesh.rachapalli.bandi@intel.com>
---
 drivers/gpu/drm/xe/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xe/Kconfig b/drivers/gpu/drm/xe/Kconfig
index 63c7b4d..6a3b974 100644
--- a/drivers/gpu/drm/xe/Kconfig
+++ b/drivers/gpu/drm/xe/Kconfig
@@ -5,7 +5,7 @@ config DRM_XE
 	depends on KUNIT || !KUNIT
 	depends on INTEL_VSEC || !INTEL_VSEC
 	depends on X86_PLATFORM_DEVICES || !(X86 && ACPI)
-	depends on PAGE_SIZE_4KB || COMPILE_TEST || BROKEN
+	#depends on PAGE_SIZE_4KB || COMPILE_TEST || BROKEN
 	select INTERVAL_TREE
 	# we need shmfs for the swappable backing store, and in particular
 	# the shmem_readpage() which depends upon tmpfs
-- 
2.43.0

